frontend-beeyond/src/app/app.module.ts

```diff
@@ -20,7 +22,15 @@ import { MatProgressBarModule } from '@angular/material/progress-bar';
     MonacoEditorModule.forRoot(),
     MatProgressBarModule
   ],
-  providers: [],
+  providers: [
+    {
+      provide: APP_INITIALIZER,
+      useFactory: (auth: AuthenticationService, config: ConfigService) => () =>
+        config.init().then(() => auth.initializeLogin().then()),
```

Comment:
`auth.initializeLogin().then()` -> `auth.initializeLogin()`
This factory method expects a Promise to be returned, so the `.then()` at the end has no meaning and can be confusion